lib/renderer.js

```diff
@@ -1142,6 +1151,10 @@
                 pixmapSettings.interpolationType = this._interpolationType;
             }
             
+            if (component.hasOwnProperty("renderOnWorkerThread")) {
+                pixmapSettings.thread = !!component.renderOnWorkerThread;
+            }
+            
```

Comment:
That was for performance, but it turns on progress dialogs can popup in some cases. Since we aren't going to use it I'm going to pull it for now. If we can safely use it in the future we will